internal/bminventory/inventory_test.go

```diff
@@ -8169,120 +7530,43 @@ var _ = Describe("V2ClusterUpdate cluster", func() {
+				})
 
-					reply := bm.V2UpdateCluster(authCtx, installer.V2UpdateClusterParams{
+				It("Update platform=none - failure", func() {
+					reply := bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
 						ClusterID: clusterID,
 						ClusterUpdateParams: &models.V2ClusterUpdateParams{
-							Platform: &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeOci)},
+							Platform: &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeNone)},
 						},
 					})
-					Expect(reply).Should(BeAssignableToTypeOf(installer.NewV2UpdateClusterCreated()))
-					actual := reply.(*installer.V2UpdateClusterCreated).Payload
-					Expect(swag.BoolValue(actual.UserManagedNetworking)).To(Equal(true))
-					Expect(*actual.Platform.Type).To(Equal(models.PlatformTypeOci))
-					Expect(swag.BoolValue(actual.Platform.IsExternal)).To(BeTrue())
+					verifyApiErrorString(reply, http.StatusBadRequest, "Can't set none platform with user-managed-networking disabled")
 				})
 
-				It("Update platform=oci - PlatformOci capability is not set - failure", func() {
-					mockOcmAuthz.EXPECT().CapabilityReview(gomock.Any(), payload.Username, ocm.PlatformOciCapabilityName, ocm.OrganizationCapabilityType).Return(false, nil).Times(1)
-
-					reply := bm.V2UpdateCluster(authCtx, installer.V2UpdateClusterParams{
+				It("Update UMN=false and platform=none results BadRequestError - failure", func() {
+					reply := bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
 						ClusterID: clusterID,
 						ClusterUpdateParams: &models.V2ClusterUpdateParams{
-							Platform: &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeOci)},
-						},
+							Platform:              &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeNone)},
+							UserManagedNetworking: swag.Bool(false)},
 					})
-					verifyApiErrorString(reply, http.StatusBadRequest, "Platform oci is not available")
+					verifyApiErrorString(reply, http.StatusBadRequest, "Can't set none platform with user-managed-networking disabled")
 				})
 
-				It("Update platform=oci - CapabilityReview returns an error - failure", func() {
-					mockOcmAuthz.EXPECT().CapabilityReview(gomock.Any(), payload.Username, ocm.PlatformOciCapabilityName, ocm.OrganizationCapabilityType).Return(true, errors.Errorf("some error")).Times(1)
+				It("Update UMN=true and platform=none - success", func() {
+					mockSuccess()
+					mockProviderRegistry.EXPECT().SetPlatformUsages(commontesting.EqPlatformType(models.PlatformTypeNone), gomock.Any(), mockUsage)
 
-					reply := bm.V2UpdateCluster(authCtx, installer.V2UpdateClusterParams{
+					reply := bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
 						ClusterID: clusterID,
 						ClusterUpdateParams: &models.V2ClusterUpdateParams{
-							Platform: &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeOci)},
+							Platform:              &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeNone)},
```

Comment:
Why replacing Platform OCI with Platform None?

Reply:
I removed all oci tests, for some reasons the diff shows changes with none platform tests, but I did not change that.